src/Standards/BestIt/Sniffs/Formatting/EqualOperatorSniff.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BestIt\Sniffs\Formatting;
+
+use BestIt\Sniffs\AbstractSniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Class EqualOperatorSniff.
+ *
+ * @author Mika Bertels <mika.bertels@bestit-online.de>
+ * @package BestIt\Sniffs\Formatting
+ */
+class EqualOperatorSniff extends AbstractSniff
+{
+    /**
+     * Code when an equal operator was found.
+     *
+     * @var CODE_EQUAL_OPERATOR_FOUND string
+     */
+    public const CODE_EQUAL_OPERATOR_FOUND = 'EqualOperatorFound';
+
+    /**
+     * Warning when an equal operator was found.
+     *
+     * @var WARNING_EQUAL_OPERATOR_FOUND string
+     */
+    public const WARNING_EQUAL_OPERATOR_FOUND = '
+        An "Equal"-operator (==) was found. You should use the "Identical"-operator (===).
+    ';
+
+    /**
+     * Find the T_IS_EQUAL token and add a warning.
+     *
+     * @param File $phpcsFile
+     * @param int $stackPtr
+     *
+     * @return void
+     */
+    private function findToken(File $phpcsFile, int $stackPtr): void
```

Comment:
Du hast in der Klasse properties. Im letzten Sniff hatte ich das nur gemacht mit den Parameter, damit Du es leichter übertragen kannst.